src/widgets/highcharts/ui/DailyTaskActivityChart/DailyTaskActivityChart.tsx

```diff
@@ -0,0 +1,138 @@
+    ];
+
+    const {
+        dates, createdTasksCount, completedTasksCount,
+    } = useMemo(() => getDataForChart(month, year, tasks), [year, month, tasks]);
+
+    const isWeekend = (date: Date) => {
+        const day = date.getDay();
+        return day === 0 || day === 6;
+    };
+
+    const options: Highcharts.Options = useMemo(() => addColorSchemeToHighcharts({
+        title: {
+            text: t('Number of tasks created and completed'),
+        },
+        xAxis: {
+            title: {
+                text: t('Day of the month'),
+            },
+            categories: dates.map((date) => date.getDate().toString()),
+            plotBands: dates.reduce((bands: Band[], date, index) => {
+                if (isWeekend(date)) {
+                    const band = {
+                        color: 'rgba(64,238,6,0.34)',
+                        from: index - 0.5,
+                        to: index + 0.5,
+                    };
+                    bands.push(band);
+                }
+                return bands;
+            }, []),
+        },
+        yAxis: {
+            title: {
+                text: t('Number of tasks'),
+            },
+        },
+        series: [{
+            type: 'line',
+            name: t('Created tasks'),
+            data: createdTasksCount,
+        }, {
+            type: 'line',
+            name: t('Completed tasks'),
+            data: completedTasksCount,
+        }],
+        credits: {
+            enabled: false,
+        },
+    }, mode), [completedTasksCount, createdTasksCount, dates, t, mode]);
+
+    return (
+        <Box sx={styles.wrapper}>
+            <Box sx={styles.select}>
+                <FormControl>
+                    <InputLabel id="select-month">
+                        {t('Month')}
+                    </InputLabel>
+                    <Select
+                        labelId="select-month"
+                        id="select-month"
+                        value={month.toString()}
+                        label={t('Month')}
+                        onChange={handleChangeMonth}
+                    >
+                        {months.map((mon, index) => (
+                            <MenuItem key={mon} value={index}>
+                                {t(mon)}
+                            </MenuItem>
+                        ))}
+                    </Select>
+                </FormControl>
+                <FormControl>
+                    <InputLabel id="select-year">
+                        {t('Year')}
+                    </InputLabel>
+                    <Select
+                        labelId="select-year"
+                        id="select-year"
+                        value={year.toString()}
+                        label={t('Year')}
+                        onChange={handleChangeYear}
+                    >
+                        <MenuItem value={2023}>2023</MenuItem>
+                        <MenuItem value={2024}>2024</MenuItem>
```

Comment:
obviously, not a good approach for generating list of available years. it need to be build dynamically based on the current year as the latest available value. the earliest available year could be specified on agreement, it could be e.g.:
- the earliest available year of createdAt among the tasks
- agreed amount of years in the past from the current year (-10, for example)
- harcoded year of the project start

any of the mentioned options could be used here

Reply:
Done, now years are counted from the creation of the first task.